changelog/unreleased/change-update-libregraph

```diff
@@ -0,0 +1,5 @@
+Change: Update libre-graph-api to v1.0
```

Comment:
this is type `enhancement` isn't it?

Reply:
Thought the same, but vue bump is classified as change too.

https://github.com/owncloud/web/blob/master/changelog/unreleased/change-update-vue

should i change it?

Reply:
but this only adds tag management. so it's an enhancement :-)

Reply:
i change it, can we force merge? don't want to wait for a full ci run (which will be the same as the last)